packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -332,6 +343,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         };
 
         this.connectionManager = createConnectionManager(props);
+        this.lastSubmittedClientId = this.connectionManager.lastSubmittedClientId;
```

Comment:
This is constructor of DeltaManager, we know we have no connection established (or even attempted to establish) during this flow, so you can just leverage default value of undefined here and not add this line.

Reply:
That said, I do not see any place where this.lastSubmittedClientId would be updated. I.e. I believe it would stay undefined forever.